queries/java.scm

```diff
@@ -326,3 +326,43 @@
   body: (_) @value
   (#not-type? @value block)
 ) @_.domain
+
+;;!! foo(name: string) {}
+;;!      ^^^^^^^^^^^^
+(
+  (formal_parameters
+    (_)? @_.leading.endOf
+    .
+    (_) @argumentOrParameter
+    .
+    (_)? @_.trailing.startOf
+  ) @_dummy
+  (#not-type? @argumentOrParameter "block_comment")
```

Comment:
might be worth folding a comment into a test?

Reply:
Yeah probably worth discussing, but out of scope for this PR

Reply:
Agreed